packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -487,6 +487,48 @@ void main() {
     expect(renderEditable.selectPositionAtCalled, isTrue);
   });
 
+    testWidgets('TextSelectionGestureDetectorBuilder right click', (WidgetTester tester) async {
```

Comment:
nit: indentation

Reply:
Also could you add a comment to this test that says it's a regression test for https://github.com/flutter/flutter/issues/80119, for posterity?

Reply:
Thank you for the review! Both issues are now fixed.